core/src/main/java/com/scalar/db/storage/dynamo/BatchHandler.java

```diff
@@ -32,17 +35,21 @@
 public class BatchHandler {
   private final DynamoDbClient client;
   private final TableMetadataManager metadataManager;
-
+  private final String namespacePrefix;
```

Comment:
Please add a newline here.
```suggestion
  private final String namespacePrefix;

```